tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -140,6 +139,10 @@ def create_parser(product_choices=None):
                                       nargs="*", default=wpttest.enabled_tests,
                                       choices=wpttest.enabled_tests,
                                       help="Test types to run")
+    test_selection_group.add_argument("--subsuite-file", action="store",
+                                      help="Path to JSON file containing subsuite configuration")
+    test_selection_group.add_argument("--subsuite", action="append", dest="subsuites",
```

Comment:
Can devs use `--subsuite name:some/test.html` to run a specific test in a subsuite? I think this will be convenience for devs to debug a failure in that scenario.

Reply:
I realized I can make a change at chromium side, so that all the subsuites only includes the tests we actually want to run. So we wont't need a change in wptrunner.